src/towncrier/test/test_check.py

```diff
@@ -41,6 +41,15 @@ def commit(message):
     call(["git", "commit", "-m", message])
 
 
+def stage():
+    """Stage a commit to the repo in the current working directory
+
+    There must be uncommitted changes otherwise git will complain:
+    "nothing to commit, working tree clean"
+    """
+    call(["git", "add", "."])
```

Comment:
Shouldn't this check the exit status?

I suppose the same applies to all the other helper functions here. You needn't fix those, but it would be appreciated.

Reply:
Ok, switched all `call` commands to `check_call` so an exception is thrown if a call fails

5b5c8ca5b5438dc5db8d212becc068b1effe56c7

Reply:
Thank you for this!